integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -74,4 +113,192 @@ void testDropSchema() {
     Assertions.assertThrowsExactly(
         NoSuchNamespaceException.class, () -> sql("drop database notExists"));
   }
+
+  // Create simple tables without advanced features like comment, property, partition, bucket.
+  // To check whether create and load operation with default.
+  @Test
+  void testCreateSimpleTable() {
+    String tableName = "simpleTable";
```

Comment:
by default spark is not case-sensitive,   let me check the overall logic to fix.

Reply:
SparkSQL itself doesn't care case-sensitive, but the underlying catalog cares. Hive is not case-sensitive, JDBC is case-sensitive. For spark-connector I think there's no need to handle case-sensitive issues. just need to pass through the name from the underlying catalog to spark SQL.  do you agree?

Reply:
For this cases, spark-connector pass origin name not the name from Gravitino back to SparkSQL, so the IT passes. I think we should pass the name from Gravitino to SparkSQL.

Reply:
So does here really test from end to end, what will the actual table be created in Gravitino? If it is a hive table, does it really support table name like "simpleTable"?

Reply:
yes, the test is from end to end,  take create hive table SimpleTable for example,
1. spark SQL pass create table request to spark-connector, with identifier `SimpleTable`
2. spark-connector request create table to Gravitino with identifier `SimpleTable`
3. Gravitino creates hive table with identifier  `SimpleTable`
4. Hive metastore create table with `simpletable`, because it's not case sensitive
Load  table process:
6. Spark SQL load table `SimpleTable` ,  spark connector request Gravitino with this identifier too.
7.  The identifier returned by Gravitino is `simpletable`, but is not used by spark-connector
8. spark connector pass origin `SimpleTable` to SparkSQL, so the test check passes.

we should pass the identifier returned by Gravitino to keep consistent with SparkSQL origin actions.